src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -99,6 +144,27 @@
     </div>
     <!-- end refresh button -->
   </div>
+  <div class="dataTables_header clearfix"
+       *ngIf="toolHeader && columnFiltered">
+    <div class="filter-chips">
+      <span *ngFor="let filter of columnFilters">
+        <span *ngIf="filter.value"
+              class="badge badge-info mr-2 rounded-0">
```

Comment:
```suggestion
              class="badge badge-info mr-2">
```
angled:
![Bildschirmfoto vom 2020-01-09 13-02-19](https://user-images.githubusercontent.com/1897962/72066381-b50fdc00-32e0-11ea-9a5c-b20d8c1281cd.png)

rounded:
![Bildschirmfoto vom 2020-01-09 13-03-05](https://user-images.githubusercontent.com/1897962/72066389-bb9e5380-32e0-11ea-836a-897e4aec481a.png)

![Bildschirmfoto vom 2020-01-09 13-03-22](https://user-images.githubusercontent.com/1897962/72066393-bf31da80-32e0-11ea-9e37-b7b2c828573d.png)

![Bildschirmfoto vom 2020-01-09 13-04-25](https://user-images.githubusercontent.com/1897962/72066397-c2c56180-32e0-11ea-9e62-de880fe9b5e7.png)

Reply:
Thanks. Changed!